app/helpers/users_helper.rb

```diff
@@ -1,4 +1,17 @@
 # frozen_string_literal: true
 
 module UsersHelper
+  def last_admin?(user)
+    user.admin? &&
+      current_user?(user) &&
```

Comment:
Why do you need that? Does the function `last_admin?` really want to check if the given user is the current user?

Reply:
That was the plan. The requirement is that an admin is the only one that can change roles and that he can not change his own role if the is the last admin left. I will split it up :+1: